### PR TITLE
Add newsletter form to nothing-personal page (Fixes #15218)

### LINKED ISSUE #15218: [nothing personal] Add newsletter form
## Description

- Ask is to add a newsletter form to the bottom of this page: https://www.mozilla.org/en-US/firefox/nothing-personal/
- Include a page anchor link that will enable linking directly to the newsletter section.
- The sign-up form should have two check boxes – one for the Fx newsletter, another for opting in to receive college related news.

See the [brief here](https://docs.google.com/document/d/13xZqrLeQ2sKKlHzEjctmecDYS3aA9NEm565Y2NMw67M/edit?usp=sharing) for more details.

---

#### Success Criteria
- [ ] Landing page has signup form for Fx News / college related news.

### PR DESCRIPTION
**Pull request by alexgibson**
## One-line summary

Adds a newsletter form to the bottom of `/firefox/nothing-personal/`.

## Issue / Bugzilla link

#15218

## Testing

http://localhost:8000/en-US/firefox/nothing-personal/

### CODE REVIEW

**Review by reemhamz — APPROVED** [review]
Looks great, works great. r+
I'll leave this for you to merge when you get the final go-ahead 👍🏼

File: bedrock/newsletter/forms.py [hunk 1]
```diff
@@ -189,9 +189,10 @@ class NewsletterFooterForm(forms.Form):
 
     # has to take a newsletters argument so it can figure
     # out which languages to list in the form.
-    def __init__(self, newsletters, locale, data=None, *args, **kwargs):
+    def __init__(self, newsletters, locale, data=None, multi_opt_in=False, *args, **kwargs):
```
[the remark this answers is not in the document]
Comment (stevejalim) [reply bedrock/newsletter/forms.py:192]: Or maybe renaming it to `multi_opt_in_required` or similar?

**Review by stevejalim — APPROVED** [review]
Makes sense, but it took a moment to wrap my head around it. Maybe some comment or tweaking a var name would help our future selves?

r+wc

File: bedrock/firefox/templates/firefox/nothing-personal/index.html [hunk 2]
```diff
@@ -192,9 +192,10 @@ <h4><span aria-hidden="true">[</span>Sign up for our newsletter<span aria-hidden
               include_language=False,
               include_country=False,
               newsletters='mozilla-and-you, nothing-personal-college-interest',
+              multi_opt_in=True
             )}}
 
-            <p>By subscribing, you’ll receive Mozilla updates, along with exclusive college-related content. You may unsubscribe at any time.</p>
+            <p>By subscribing, you’ll receive Mozilla updates, plus exclusive college-related content if you opt in. You may unsubscribe from either at any time.</p>
```
[the remark this answers is not in the document]
Comment (alexgibson) [reply bedrock/firefox/templates/firefox/nothing-personal/index.html:198]: That's correct yeah

**Pull request merged**